save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/FossGraphView.kt

```diff
@@ -111,6 +113,15 @@ val fossGraph: FC<FossGraphViewProps> = FC { props ->
             }
 
         setVulnerability(vulnerabilityNew)
+
+        val userInfo = get(
```

Comment:
```suggestion
        val userInfo: UserInfo = get(
```
and remove suppressing

Reply:
Fixed